### ISSUE TITLE
Can't upgrade KSC facilities

### ISSUE THREAD

**Issue by DanielKinsman** [magico13/KCT#112]
KCT tells me (on the tech tab) that "No KSC upgrade projects are currently underway". Yet when I try to upgrade the launchpad (KSP upgrade, not KCT upgrade) it tells me "Facility is already being upgraded" and aborts the process.

The building upgraded instantly once I turned off "KSC Upgrade Times" in the settings, so perhaps it was being upgraded but the entry was missing from the KCT tech tab.

This is on KSP 1.0.5, KCT 1.2.3, realism overhaul 10.6.0 and a dev version of RP-0.

**Comment by magico13** [comment]
You're using RO, are you also using KSCSwitcher? (that's the mod that lets you move the KSC to different places, so you can launch from real sites). If so, did you go into the Tracking Station and make sure to select a launch site through there before building anything?

I never tested KSC upgrades with KSCSwitcher. It's possible that they're in a weird half-state where they're tied both to a particular KSC (and so will only show up in that KSC's Tech List) but also tied globally so that the actual upgrade state is shared. If that's the case then you'd have to figure out which KSC is upgrading it to see it in the list. It should still be built in the background and once it's done it would update globally. A related issue (#97) would make each KSC have their own unique upgrades, which would fix the half-state issue.

**Comment by DanielKinsman** [comment]
Yep, using switcher. Nope, never selected a site in the tracking station.

**Comment by magico13** [comment]
It could be stuck in the "Stock" KSC or something like that then (which unfortunately can't be accessed again when using KSC Switcher). I'll test it and see what I can find. If you see that crop up again you might just have to turn off the upgrade times. Alternatively I think the "Warp To" button warps to the next thing to be built no matter which KSC it's at.

**Comment by ShloEmi** [comment]
I've turned the RP mod on, warped time till Launch-pad was constructed, and disabled it again, it works.
HTH someone.
